api/proto/metrics/metrics.proto

```diff
@@ -93,6 +94,16 @@ message deviceMetric {
   logMetric log = 12;
 }
 
+message containerMetric {
+  string containerName = 1;   // the unique key for the container in a VM or IoT Edge
```

Comment:
Shouldn't we also have some reference to the appInstance UUID in which the container/module is running?
In case there is some other VM which is running some other app instance.

Reply:
this containerMetric{} is one item inside the appMetric{} which has the 'AppID', so this is already per App, or per VM.

Reply:
Ah - I missed that.